src/game/mod.rs

```diff
@@ -606,7 +607,6 @@ impl GameState {
 	fn draw_map(&mut self, engine: &mut Engine<GameState>) -> anyhow::Result<()> {
 		if self.tiles_meshes.is_empty() {
 			let mut mesh_builders: Vec<_> = (0..self.tiles_atlas.len_atlases())
-				.into_iter()
```

Comment:
Also old leftovers from when I was originally building an iterator, this is indeed unneeded for ranges.  ^.^